prow/cmdutil/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package cmdutil contains utilities and interfaces shared between
+// several test-infra commands.
+package cmdutil
```

Comment:
> In reality this package just contains logic to help initialize various clients.

So... `clientinit`?  I wsa thinking of this as "utilities for our command-line programs" (hence my current name), and I like having `prow/cmdutil` as the Prow-specific analog to `cmdutil` (where I'm putting the not-Prow-specific `OptionGroup` interface and `ValidateURIs` helper.  Do you want me to rename that to `clientinit` (or whatever) as well?

Reply:
Yes, I think both should be renamed.
I think this could be suitable to add to the [`flagutil` package](https://github.com/kubernetes/test-infra/tree/master/prow/flagutil) (rename flagutil.go to strings.go and add a new file in the same package). Replacing "cmd" with "flag" distinguishes this from other types of utilities that a binary might need and from Prow commands like `/lgtm`. WDYT?
The non-Prow-specific package could also be called `flagutil`?

Reply:
> I think this could be suitable to add to the flagutil package...

Done with b9c239d -> 983c79c.